content/docs/ops/runbook/troubleshooting-snort.md

```diff
@@ -0,0 +1,29 @@
+---
+menu:
+  docs:
+    parent: runbook
+
+title: Troubleshooting Snort
+---
+
+## Overview
+Snort is a network intrusion detection system that runs on all cloud.gov hosts.
+
+### Responding to snort alerts
+* Identify the rule that triggered the alert. Snort alerts include a snort rule ID (sid), such as `1:22968`. Check https://www.snort.org/rule_docs/ for a description of the alert rule and its impact and mitigations.
+* Check the snort logs to verify that the alert is a true positive.
+    * Connect to the host identified in the error message.
+    * Inspect the snort logs:
+
+        ```
+        cat /var/vcap/sys/log/snort-eth0/snort.log.XXXXX | strings
+        ```
+
+    * Optionally use `u2spewfoo` to decode the snort binary format:
+        
+        ```
+        /var/vcap/packages/snort/bin/u2spewfoo /var/vcap/sys/log/snort-eth0/snort.log.XXXXX
+        ```
+
+* If the alert appears to be a false positive, consider excluding the rule from the snort configuration.
```

Comment:
Can we link to something that would tell someone how to exclude false positives from the rule set and guidelines for doing so? I wouldn't know where to do that today as https://github.com/18F/cg-snort-boshrelease  / https://github.com/18F/cg-snort-boshrelease/blob/master/jobs/snort/spec is effectively undocumented.

We might also want to touch on what VMs an exclusion should cover?
Should it be done on just the job vms affected? All VMs in the deployment? All of cloud.gov?